src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -11,7 +11,10 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Test;
 
+use Symfony\Bundle\FrameworkBundle\Tests\Functional\Kernel\TestKernel;
+use Symfony\Bundle\FrameworkBundle\Tests\Functional\Kernel\TestKernelInterface;
```

Comment:
no, they definitely are not required for correct work of KernelTestCase. Moved them to Test namespace